src/pages/Build/TermsOfService.vue

```diff
@@ -0,0 +1,194 @@
+<template>
+	<www-page>
+		<developer-secondary-menu slot="secondary" />
+		<div class="row page-content">
+			<div class="small-12 columns">
+				<h1>Terms Of Service</h1>
+				<p>Last updated: February 19, 2013</p>
+				<h1 class="text-right">KIVA DEVELOPER TERMS OF SERVICE</h1>
```

Comment:
Did the "text-center" class not work...

Also, the "text-right" class will right alight text so we shouldn't use it here unless you want right aligned text...and it's super confusing to set a class saying right then apply a center style to it.

Reply:
ok. thanks for the guidance! I'll try it with "text-center" without the custom css at the bottom. Appreciate your 👀 on this!